Day23 Project - Simple Calculator (Issue 252)

chhaviluthra08 opened the pull request:
### Added a new Calculator project [Resolves ISSUE-252]

This PR addresses the newly added calculator project for the 100Days100WebProjects, the Calculator is stylish it includes 3-d buttons that work with 3-d effects.

### Project Screenshot and Video 

https://github.com/user-attachments/assets/a894c0a1-2497-440e-80d4-774234621c21

<img width="995" height="750" alt="Screenshot 2025-07-29 at 7 54 55 PM" src="https://github.com/user-attachments/assets/d421df9c-ecb5-431b-82cb-22a5b5a14ac9" />

### 📁 Files and Folders Modified

* `./index.js`
* `./public`

__Please let me know if any further changes are needed__

Laharivanja commented:
Hey @chhaviluthra08! Can you please let me know how you managed to select the day as 2-3 people are using the same day and leading to merge conflicts?

chhaviluthra08 commented:
Hey @Laharivanja, I just made my project first without any forking to ensure no conflicts occur when I'm ready to raise a pr, then I forked the repo and based on the last largest day I changed my file's name to the day after that (for eg: at my time the previous largest day was day 22 so I simply changed my project folder's name to day 23). Then I made the changes in the new branch and pushed all the changes after committing and then I raised a pr. 
If you were working on a specific day and somehow someone else has also added a project on that day, simply add your project as another day just like I explained.
Hope this clears your doubt, if not you can ask me again :))

Laharivanja commented:
@chhaviluthra08 Thanks alot☺️!!

chhaviluthra08 commented:
@Laharivanja I'm not sure why that's happening it's looking okay on the wesbite.
